tests/unit/FilingHistoryList.spec.ts

```diff
@@ -1703,7 +1703,16 @@ describe('Filing History List - corrections', () => {
           'correction': {
             'correctedFilingType': 'annualReport',
             'correctedFilingDate': '2019-12-31'
-          }
+          },
+          documents: [
```

Comment:
You should add some "expects" to verify this document... otherwise what is the purpose of having it here?

Reply:
Right. Will do it.

Reply:
Awesome, thanks!